app/model/editions/templates/TrainingEdition.scala

```diff
@@ -14,6 +14,7 @@ object TrainingEdition extends InternalEdition {
   override val header = Header("Training Edition")
   override val notificationUTCOffset = 3
   override val topic = "tr"
+  override val locale = Some("tr_TR")
```

Comment:
I'm concerned this might prevent us viewing the Training Edition on devices.  May I suggest en_GB instead?